recipes/affine/index.js

```diff
@@ -0,0 +1,6 @@
+module.exports = Ferdium =>
+  class Affine extends Ferdium {
+    buildUrl(url) {
+      return `${url}/`;
```

Comment:
should this be more defensive and check if the url ends with a `/` before appending?

Reply:
I started from the Firefly recipe which I made earlier which has this implemented. Is the `/` required? Otherwise I will omit this later today.

Reply:
Removed the `/`, tested and works fine